workflows/models.py

```diff
@@ -1017,9 +1017,15 @@ def generate_inputs_yaml(self, inputs_yaml_filename):
             if dataset_class not in input_info:
                 input_info[dataset_class] = {}
 
-            if sample_library_id not in input_info[dataset_class]:
+            if library_type == 'WGS':
                 input_info[dataset_class][sample_library_id] = {}
```

Comment:
Does WGS still need sample_id + '_' + library_id?  I think everythings changed to sample then library, check if you could